Changes.md

```diff
@@ -3,6 +3,11 @@ Improvements
 
 - Merge : Optimize image merging.  This has been tackled in several ways, with different levels of impact.  Some extreme cases, such as using a multiply to merge two large datawindows with little overlap, now produce much smaller data windows.  Other cases can benefit a lot from being able to pass through input tiles unmodified.  For cases without a huge shortcut, there is an approximately 20% speedup from lower level optimization.
 
+Breaking Changes
+----------------
+
+- Checkerboard : Checker colors are now exactly equal to the colorA and colorB parameters.  Previously, there were very tiny floating point errors which grew larger as the distance from origin increased.
+
```

Comment:
I think we can classify this as a fix rather than a breaking change. I don't picture anyone relying on the floating point errors.